cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -304,18 +304,17 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     )
   }
 
-  //TODO reinstate when "Composite lookups not yet supported" bug is fixed
-//  @Test def break_node_key_constraint_by_removing_property() {
-//    generateConsole = false
-//    execute("CREATE CONSTRAINT ON (n:Person) ASSERT (n.firstname, n.surname) IS NODE KEY")
-//    execute("CREATE (p:Person {firstname: 'John', surname: 'Wood'})")
-//    testFailingQuery[ConstraintValidationException](
-//      title = "Removing a `NODE KEY` constrained property",
-//      text = "Trying to remove the `surname` property from an existing node `Person`, given a `NODE KEY` constraint on `:Person(firstname, surname)`.",
-//      queryText = "MATCH (p:Person {firstname: 'John', surname: 'Wood'}) REMOVE p.surname",
-//      optionalResultExplanation = "In this case the property is not removed."
-//    )
-//  }
+  @Test def break_node_key_constraint_by_removing_property() {
```

Comment:
Is this used anywhere in the docs?

Reply:
Yes, see in Constraints section: "Removing a `NODE KEY`-constrained property"